src/rendervulkan.hpp

```diff
@@ -45,8 +60,9 @@ struct Composite_t
 		vec2_t vScale[k_nMaxLayers];
 		vec2_t vOffset[k_nMaxLayers];
 		float flOpacity[k_nMaxLayers];
-		float flBorderAlpha[k_nMaxLayers];
 	} data;
+
+	uint32_t nBorderMask;
```

Comment:
why is this outside of `data`?

Reply:
Moved this into data to not conflict with your changes and such.